views/MainWindow.qml

```diff
@@ -0,0 +1,127 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.4
+import QtQuick.Dialogs 1.0
+import QtQuick.Layouts 1.0
+
+ApplicationWindow {
+    id:  mainWindow
+    objectName: "mainWindow"
+    title: "MOCOS"
+    height: 500
+
+    visible: true
+
+    Component.onCompleted: {
+        mainWindow.width = initialConditionsButton.width + generalSettingsButtons.width +
+                contactTrackingSettingsButton.width + transmissionSettingsButton.width +
+                modulationSettingsButton.width + phoneTrackingSettingsButton.width + 50;
+    }
+
+    FileDialog {
+        id: projectSaveDialog
+        folder: shortcuts.home
+        selectExisting: false
+        sidebarVisible: true
+        nameFilters: [ "JSON files (*.json)" ]
+        onAccepted: {
+            projectHandler.saveAs(projectSaveDialog.fileUrl)
+        }
+    }
+
+    FileDialog {
+        id: populationFileOpenDialog
+        folder: shortcuts.home
+        selectExisting: true
+        sidebarVisible: true
+        nameFilters: [ "JLD2 files (*.jld2)" ]
+        onAccepted: {
+            generalSettings.populationPath = populationFileOpenDialog.fileUrl
+        }
+    }
+
+    menuBar: MenuBar {
+        Menu {
+            title: "&File"
+            MenuItem { text: "Load &population file"; onTriggered: populationFileOpenDialog.visible = true }
+            MenuItem { text: "&Save As..."; onTriggered: projectSaveDialog.visible = true }
+            MenuItem { text: "&Close"; onTriggered: mainWindow.close() }
+        }
+    }
+
+    header: ToolBar {
```

Comment:
it would be great if there is a key shortcut to navigate through toolbar (ctrl+tab)